sike-java/src/main/java/com/wultra/security/pqc/sike/util/OctetEncoding.java

```diff
@@ -44,7 +46,7 @@ public static String toOctetString(BigInteger n, int length) {
         char[] chars = hex.toCharArray();
         int expectedLength = length * 2;
         if (chars.length > expectedLength) {
-            throw new InvalidParameterException("Number is too large, length: " + chars.length + ", expected: " + expectedLength);
+            throw new InvalidParameterException(Constants.Exceptions.NUMBER_TOO_LARGE + ", length: " + chars.length + ", expected: " + expectedLength);
```

Comment:
Would something like this be ok ? :
``` java
  MessageFormat form = new MessageFormat("{0}, length: \"{1}\", expected: {2}.");
  Object[] args = {Constants.Exceptions.INVALID_SECRET, chars.length, expectedLength};
  throw new InvalidParameterException(form.format(args));
```

I've never really used MessageFormat.

Reply:
@kasahity In situations like this, I usually look at what the JDK team does to see if what I do is "an improvement" or just "a change". JDK team uses inline Strings when throwing exceptions, see: https://github.com/openjdk/jdk/search?q=throw+new

So I think that one option is to fully externalize Strings and use `MessageFormat` like this:

```java
// Constants.Exceptions.NUMBER_TOO_LARGE = "Number is too large, length: {1}, expected: {2}."
throw new InvalidParameterException(
    MessageFormat.format(Constants.Exceptions.NUMBER_TOO_LARGE , new Object[] {
        chars.length, expectedLength
    })
);
```

The other option is to revert this approach back to inline strings (which would probably be my choice since I am not sure that externalizing exception messages improves code readability):

```java
throw new InvalidParameterException(
    "Number is too large, length: " + chars.length + ", expected: " + expectedLength
);
```